lib/util/template/index.ts

```diff
@@ -169,6 +170,10 @@ const allowedTemplateFields = new Set([
 
 const compileInputProxyHandler: ProxyHandler<CompileInput> = {
   get(target: CompileInput, prop: keyof CompileInput): unknown {
+    if (prop === 'env') {
+      return target[prop];
+    }
```

Comment:
@zharinov does this look right?

Reply:
was my idea to simplify things.

Reply:
Can something go wrong because of this structure being nested? If not, then it looks okay to me.

Reply:
`env` should be a simple string dictionary

Reply:
I mean, the whole data structure allows for nested `env`. To the another hand, seems like it's being attached only on top level. Well, everything looks okay to me here.